components/installer/scripts/extract-commit-hash.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+
+NUMBER_OF_PARENTS=$(git show -q --format=%P | wc -w)
+
+if [ $NUMBER_OF_PARENTS -eq 1 ]
+then
+    echo $(git log -1 --format=%H)
+elif [ $NUMBER_OF_PARENTS -eq 2 ]
+then
+    echo $(git rev-parse HEAD^2)
+else
+    echo There\'s more than two parents of commit $NUMBER_OF_PARENTS.
```

Comment:
My intention was to make this message more clear.

IS (BAD, because it implies that this is third commit somehow):
There\'s more than two parents of commit 3.

SHOULD BE:
There\'s more than two parents of commit (3).

YET BETTER:
There are 3 parents of commit.